src/legacy/core_plugins/input_control_vis/public/components/editor/list_control_editor.test.js

```diff
@@ -325,7 +325,13 @@ test('field name change', async () => {
   // ensure that after async loading is complete the DynamicOptionsSwitch is disabled, because this is not a "string" field
   expect(component.find('[data-test-subj="listControlDynamicOptionsSwitch"][disabled=true]')).toHaveLength(0);
   await update();
+
+
+  /*
+  The issue with enzyme@3.9.0 -> the fix has not been released yet -> https://github.com/airbnb/enzyme/pull/2027
+  TODO: Enable the expectation after the next patch released
```

Comment:
This module isn't being worked on a ton right now, would you mind adding something to your calendar to return to this in a couple weeks once the enzyme-adapter is updated?

Reply:
@spalger sure.
@sulemanof is on vacation, so I will track this.